### PR TITLE
refactor: adjust readme after remove labeler

### PR DESCRIPTION
**Pull request by bwanglzu**
1. removed most of the outdated contents from readme and make it clean & simple
2. removed 2 tutorials rely on labeler.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,153 +45,59 @@ Requires Python 3.7+ and *one of* [PyTorch](https://pytorch.org/)(>=1.9) or [Ten
 pip install finetuner
 ```
 
-## [Documentation](https://finetuner.jina.ai)
```
Comment (tadejsv) [inline README.md:48]: Why remove link to documentation?

File: docs/get-started/celeba.md [hunk 2]
```diff
@@ -1,114 +0,0 @@
-# Finetuning Pretrained ResNet for Celebrity Face Search
```
Comment (tadejsv) [inline docs/get-started/celeba.md:1]: I beleive we then need to remove this from TOC as well. And should't we replace these tutorials with something, not just remove them? For celebrity we could fine tune with tuner, and use docarray with image sprites/embedding visualization, as well as maybe evaluator

File: README.md (hunk 1 above)
Comment (bwanglzu) [reply README.md:48]: i think everyone is searching docs on top right like this: 
0019DEA6-61C5-4589-BB7F-09F241CEDAAA @tadejsv

File: docs/get-started/celeba.md (hunk 2 above)
Comment (bwanglzu) [reply docs/get-started/celeba.md:1]: i think we have too many image related tutorials now, i think 1 image, 1 text and 1 pointcloud should be good

**Review by gmastrapas — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -45,153 +45,60 @@ Requires Python 3.7+ and *one of* [PyTorch](https://pytorch.org/)(>=1.9) or [Ten
...
-
-## Finetuning ResNet50 on CelebA
-
-> ⚡ To get the best experience, you will need a GPU-machine for this example. For CPU users, we provide [finetuning a MLP on FashionMNIST](https://finetuner.jina.ai/get-started/fashion-mnist/) and [finetuning a Bi-LSTM on CovidQA](https://finetuner.jina.ai/get-started/covid-qa/) that run out the box on low-profile machines. Check out more examples in [our docs](https://finetuner.jina.ai)!
-
+## Finetuning ResNet18 on CelebA
 
 1. Download [CelebA-small dataset (7.7MB)](https://static.jina.ai/celeba/celeba-img.zip) and decompress it to `'./img_align_celeba'`. [Full dataset can be found here.](https://drive.google.com/drive/folders/0B7EVK8r0v71pWEZsZE9oNnFzTm8?resourcekey=0-5BR16BdXnb8hVj6CNHKzLg)
-2. Finetuner accepts Jina `DocumentArray`/`DocumentArrayMemmap`, so we load CelebA image into this format using a generator:
+2. Finetuner accepts docarray `DocumentArray`, so we load CelebA image into this format using a generator:
```
Comment (makram93) [inline README.md:51]:
```suggestion
2. Finetuner accepts `DocumentArray` from the [DocArray](https://github.com/jina-ai/docarray) library, so we load CelebA image into this format using a generator:
```

File: README.md [hunk 4]
```diff
@@ -45,153 +45,60 @@ Requires Python 3.7+ and *one of* [PyTorch](https://pytorch.org/)(>=1.9) or [Ten
...
-            -1, 0
+    data = DocumentArray.from_files('img_align_celeba/*.jpg')
+    
+    
+    def preproc(doc):
+        return (
+            doc.load_uri_to_image_tensor(224, 224)
+            .set_image_tensor_normalization()
+            .set_image_tensor_channel_axis(-1, 0)
         )  # No need for changing channel axes line if you are using tf/keras
```
Comment (makram93) [inline README.md:64]:
```suggestion
        )  # No need for changing channel axis if you are using tf/keras
```

File: README.md [hunk 5]
```diff
@@ -45,153 +45,60 @@ Requires Python 3.7+ and *one of* [PyTorch](https://pytorch.org/)(>=1.9) or [Ten
...
+    
+    
+    data.apply(preproc)
     ```
 3. Load pretrained ResNet50 using PyTorch/Keras/Paddle:
```
Comment (makram93) [inline README.md:69]:
```suggestion
3. Load pretrained ResNet18 using PyTorch/Keras/Paddle:
```

File: README.md [hunk 6]
```diff
@@ -45,153 +45,60 @@ Requires Python 3.7+ and *one of* [PyTorch](https://pytorch.org/)(>=1.9) or [Ten
...
-        freeze=True,
+        loss='TripletLoss',
+        epochs=20,
+        device='cuda',
+        batch_size=128,
         to_embedding_model=True,
         input_size=(3, 224, 224),
-        layer_name='my_embedding_layer',
-        freeze=['layer_1', 'layer_2'],
+        layer_name='adaptiveavgpool2d_67', # layer before fc as feature extractor
```
Comment (makram93) [inline README.md:98]:
```suggestion
        layer_name='adaptiveavgpool2d_67', # layer preceding classification layer to be used as feature extractor
```

**Review by makram93 — COMMENTED** [review]
Some minor comments
